ops/run.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+echo -e "\nGlobally installed Botpress version: $(botpress --version)\n"
+
+if [ ! -z ${BP_PLUGIN_INSTALL+x} ];
+then
+  echo "Variable should be called BP_PLUGINS_INSTALL - plural pluginS."
+  exit 1
+fi
+
+if [ ! -z ${BP_PLUGINS_INSTALL+x} ];
+then
+  OIFS=$IFS
+  IFS=','
+
+  for x in $BP_PLUGINS_INSTALL
+  do
+    echo "Installing plugin: ${x}"
+    botpress install ${x}
```

Comment:
This command does not exist anymore (or will be removed if not). Let's use `npm install --save` instead

Reply:
Could you please provide link where in docs it is or just give a full length example for any plugin?

Reply:
> Let's use npm install --save instead

The `--save` flag is no longer needed. It's the default for npm now.